setup_lacework_agent.sh

```diff
@@ -62,6 +64,11 @@ render_agent_config() {
     _server_url_json='"serverurl": "'$SERVER_URL'",'
   fi
 
+  # Additional Config Json
+  if [ "$ADDITIONAL_CONFIG" != "" ]; then
+    _additional_config_json='$ADDITIONAL_CONFIG,'
```

Comment:
I just found this in our logs:
```
time="2023-11-02T20:54:57.451Z" level=error msg="Cannot parse config file [/var/lib/lacework/config/config.json] unable to parse content: [invalid character '$' looking for beginning of object key string]" caller="json.go:325" pid=27849
```

The cause:
```
root@ip-10-6-10-92:/var/log/lacework# cat /var/lib/lacework/config/config.json
{
  "tokens": { "AccessToken": "Nope" },

  $ADDITIONAL_CONFIG,
  "tags": {"account_id":"none","env":"dev","region":"us-east-1"}
}
```

Basically, it looks like we are rendering `$ADDITIONAL_CONFIG,` in the file, which is invalid.

Any idea?

Reply:
Hey @JPLachance taking a look at this now

Reply:
Fix PR -> https://github.com/lacework/terraform-aws-ssm-agent/pull/63